src/plots/mapbox/mapbox.js

```diff
@@ -549,7 +558,7 @@ proto.updateFx = function(fullLayout) {
     var dragMode = fullLayout.dragmode;
     var fillRangeItems;
 
-    if(dragMode === 'select') {
+    if(rectMode(dragMode)) {
```

Comment:
Why did this change? Wouldn't drawing modes just do nothing on mapbox? Same question on line 601 and a couple places in ternary.

Reply:
These plot types support `select` & `lasso`.
Drawing on `ternary` still needs a little offset fix somewhere which if out of scope; but drawing on the `mapbox` works (rather useful for highlighting some areas) i.e. in paper coordinate.